src/components/Events.module.scss

```diff
@@ -15,8 +17,22 @@
   width: 100%;
   gap: 30px;
   justify-content: space-between;
+  max-width: 100%;
 }
 
 .sport {
   font-weight: bold;
 }
+
+@media only screen and (max-width: $screen-md-min) {
+  .event {
+    margin-left: auto;
+    margin-right: auto;
+    height: 200px;
+    width: 343px;
+    border: 2px solid #333;
+    padding-top: 5px;
+    padding-left: 10px;
+    border-radius: 10px;
```

Comment:
Keep the same attributes together. Border with border, margin with padding and so on.

- https://webdesign.tutsplus.com/outside-in-ordering-css-properties-by-importance--cms-21685a